openapi/index.yaml

```diff
@@ -383,6 +412,25 @@ definitions:
     $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/master/openapi/definitions.yaml#/OrganizationFiscalCode"
   ExtendedProfile:
     $ref: "https://raw.githubusercontent.com/teamdigitale/io-functions-commons/master/openapi/definitions.yaml#/ExtendedProfile"
+  UserGroupsPayload:
```

Comment:
can we add a description and/or an example with the expected format for this array ?